src/main/java/org/sagebionetworks/web/server/servlet/filter/CrawlFilter.java

```diff
@@ -337,28 +334,24 @@ private String getDisplayName(UserProfile profile) {
     return displayNameBuilder.toString();
   }
 
-  public static String getPlainTextWiki(
-    String entityId,
-    SynapseClientImpl synapseClient
-  ) {
+  public static String getPlainTextWiki(String entityId, WikiPage rootPage) {
```

Comment:
Takes the WikiPage instead of a SynapseClientImpl.  Caller now retrieves the WikiPage